dv_utils/log_utils.py

```diff
@@ -37,6 +38,27 @@ def audit_log(log:str|None=None, data:dict|None=None, app_id:str|None=None, app=
        if(r.status_code!=204):
           print(f"Error pushing log {r}", flush=True)
 
+def audit_log_basic(log:str|dict, level:str):
```

Comment:
Waarom niet audit log vervangen met deze functie?

Reply:
omdat de signature anders is en we de bestaande dingen niet willen breken. Ik kan wel de ene functie de andere laten gebruiken.

Reply:
Hmm ja, 't is wel wat jammer natuurlijk.  Uiteindelijk zitten we nog in een 0.x versie dus breaking changes mogen wel. + de oude audit_log oproepen met de nieuwe signature gaat nog steeds werken want dat wordt overal gewoon opgeroepen met een string als parameter. Die extra parameters worden echt nergens gebruikt